QuasistaticBrownianThermalNoise/QuasistaticBrownianThermalNoise.cpp

```diff
@@ -1311,10 +1533,15 @@ void ElasticProblem<dim>::output_results(const unsigned int cycle) {
 
   const double subNoiseTimesf =
       toSI * fourKbToverPi * substrateEnergy * subLossAngle;
-  const double coatNoiseTimesf =
-      // toSI * fourKbToverPi * coatingEnergy * lossPhi_AlGaAs;
-      // Use same loss angle for coating whether crystal or effective isotropic
-      toSI * fourKbToverPi * coatingEnergy * coatLossAngle;
+
+  double coatNoiseTimesf;
+  if (config_multloss["CustomLossAngles"].as<bool>()) {
+    coatNoiseTimesf = toSI * fourKbToverPi * coatingEnergy;
```

Comment:
Add comment: if using multiple loss angles, no need to scale the result by an overall loss angle here; the correct scaling of the coating energy has already been done.